client/document.go

```diff
@@ -283,6 +283,11 @@ func (doc *Document) setAndParseType(field string, value any) error {
 		if err != nil {
 			return err
 		}
+	case uint64:
```

Comment:
Because of the json marshalling and unmashalling of the documents. Type int can be interpreted as such when imported. If I remove this case, this is the error we get: `unexpected type. Property: age, Actual: uint64`

Reply:
Okay, this needs to change though, because now you are persisting JSON parsing logic into CBOR database storage.  Lens had this issue too.

The type/values that we store in the database stores should not be dependent on external data formats.

Reply:
Can you please point me towards what you did with Lens for this situation?

Reply:
It was the following lines in `core.DecodeFieldValue`:
```
case client.FieldKind_INT:
	switch v := val.(type) {
	case float64:
		if v >= 0 {
			return uint64(v), nil
		}
		return int64(v), nil
	}
}
```
This ensured that Json-parsed values were decoded to standard (i.e. CBOR) types.

Reply:
added temporary fix until #1681 is resolved.

✅